src/app/item-popup/PlugTooltip.m.scss

```diff
@@ -59,7 +59,7 @@
     @include tooltip-section-color($red);
   }
   .automaticallyPickedSection {
-    @include tooltip-section-color($orange);
+    @include tooltip-section-color(#e8a534);
```

Comment:
Since [we're highlighting automatically picked mods with the primary accent colour](https://github.com/guise/DIM/blob/sass-to-css-variables/src/app/loadout/loadout-ui/Sockets.m.scss#L18), perhaps we should also change this to the primary accent colour?

Reply:
That's what this is, but I had to hard-code the accent colour since the mixin doesn't accept css vars as a parameter. 

It's a bit of a hack but thinking this can be fixed properly if/when it's clear if this needs to be part of the theme in future.